README.md

```diff
@@ -267,6 +267,8 @@ Use the "Table of Contents" menu on the top-left corner to explore the list.
 - [vite-plugin-solid-markdown](https://github.com/xbmlz/vite-plugin-solid-markdown) - A vite plugin for compiling markdown files to solid components.
 - [vite-plugin-qiniu-oss](https://github.com/th-come/vite-plugin-qiniu-oss) - Upload the production files bundled in the project to qiniu OSS, except for HTML.
 - [vite-plugin-stylelint](https://github.com/ModyQyW/vite-plugin-stylelint) - Runs Stylelint synchronously/asynchronously.
+- [vite-plugin-monkey](https://github.com/lisonge/vite-plugin-monkey) - A vite plugin server and build your.user.js for userscript engine like Tampermonkey, Violentmonkey and Greasemonkey
```

Comment:
## Checklist

- [ ] git: pull request template structure not broken
- [ ] git: pull request template not selected `type`
- [ ] git: pull request template not checked `Checklist`
- [ ] git: empty commit
- [ ] git: link not to repository
- [ ] git: link description contains a link to the author / third-party resource
- [x] git: link description no `for Vite` or `Vite plugin`
- [ ] git: link description no `Fast` or `Super-fast` etc
- [ ] git: link description short description short description (one sentence in 24 words, no expanded explanation)
- [ ] git: terms in the wrong formats (e.g. Typescript over TypeScript)
- [ ] git: broken PR
- [ ] git: no conflicts resolved
- [ ] git: doublicate
- [ ] time: not updated for more than 6 months
- [ ] category: not in its category
- [ ] lists: not the last on the list
- [ ] docs: no documentation
- [ ] docs: non-english documentation
- [ ] project: commercial product
- [ ] project: dead project
- [ ] project: does not accept donat
- [ ] site: no link to the site
- [ ] site: site with errors
- [ ] site: not actively using vuejs
- [ ] site: not original
> Follow the rules [CONTRIBUTING](https://github.com/vitejs/awesome-vite/blob/master/.github/contributing.md)